Go2.html

```diff
@@ -92,25 +92,98 @@
             stoneColourArray.push(0);
         };
 
+        let stoneLeft = [];
+            let stoneRight = [];
+            let stoneTop = [];
+            let stoneBottom = []; //0= not surrounded; 1 = surroudned by black; 2 = surrounded by white; 3 = surrounded by Black or White (for corners)
+            let stoneSurrounded = []; 
+
+        for (let i = 0; i < 361; i++) {
+            stoneLeft.push(0);
+            stoneRight.push(0);
+            stoneTop.push(0);
+            stoneBottom.push(0);
+            stoneSurrounded.push(0); 
+        };
+
+        for (let i = 0; i < 19; i++){
+            stoneLeft[i] = 3;
+            stoneRight[i + 342] = 3;
+            stoneTop[i * 19] = 3;
+            stoneBottom[(i * 19) + 18] = 3; //value of 3 indicates both black and white
+        };
+        
+        let whoseTurn = 2; //1 = Black 2 = White
```

Comment:
For these kinds of numbers that have meaning, it's better to declare them as a constant at the top of the file and use the constant instead. In our case, we should declare at the top
```
const BLACK = 1;
const WHITE = 2;
```
and
```suggestion
        let whoseTurn = WHITE;
```